l10n_ro_config/data/data.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
```

Comment:
Astea oricum sunt ok pentu urmatoarele companii din baza de date, pengtru ce ai deja, trebuie pus in setup la companie separat